src/main/java/org/folio/linked/data/client/SearchClient.java

```diff
@@ -16,6 +16,6 @@
 public interface SearchClient {
 
   @PostMapping(value = "/index/indices", consumes = APPLICATION_JSON_VALUE, produces = APPLICATION_JSON_VALUE)
-  ResponseEntity<FolioCreateIndexResponse> createIndex(@RequestBody CreateIndexRequest createIndexRequest);
+  ResponseEntity<CreateIndexResponse> createIndex(@RequestBody CreateIndexRequest createIndexRequest);
```

Comment:
Not in scope of this story. 
I think we do not have to create index. Mod-search should be automatically doing it.
We can create another story to look into it and remove this if not needed.

Reply:
Yep, that's true and I've also noticed that mod-search does it